src/Temra.ts

```diff
@@ -111,41 +111,45 @@ export class Temra<E = any, O = any> {
         rewriter.on(selector, deleteElement);
         continue;
       }
-      let ElementHandler: HTMLRewriterElementContentHandlers;
+      let ElementHandler: HTMLRewriterElementContentHandlers = {};
       switch(replacer.type) {
         case "universal":
         case "tagName":
           ElementHandler.element = async (element: Element) => {
+            if (!replacer.replacerFunction) return;
```

Comment:
These `replacerFunction`s would've thrown if it was called when undefined, so we at least prevent that. Again, not sure if functionally correct.